ci_framework/roles/edpm_build_images/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Install edpm-image-builder
+  ansible.builtin.include_tasks: install.yml
+- name: Run edpm-image-builder
+  ansible.builtin.include_tasks: run.yml
+- name: Package build images inside container image
+  ansible.builtin.include_tasks: package.yml
```

Comment:
Please include ci_framework/roles/edpm_build_images/tasks/cleanup.yaml playbook and add a var to enable the cleanup. By default keep it false.